willeswa/sendit-app
# #161583366 design dashboard for user

**What does this PR do?**
Have table and navigation mockup for the dashboard
**Description of Task to be completed?**
A user has the ability to check the history of placed orders and also orders that he has placed and are pending delivery
**How should this be manually tested?**
Clone this repo into your local computer. 
Navigate to the UI folder and open the user.html file. You can see the links that would be implemented at a later stage to allow for the required functionality
**Any background context you want to provide?**
Some functionality will be implemented during the implementation of the backend
**What are the relevant pivotal tracker stories?**
#161583366
**Screenshots (if appropriate)**
![screenshot from 2018-11-01 23-59-51](https://user-images.githubusercontent.com/38868860/47881040-0ea38d80-de36-11e8-9776-784277315de8.png)
